src/modules/Makefile.in

```diff
@@ -111,6 +114,16 @@ install-modules:	$(MODULES)
 		$(INSTALL) -m 0755 lua-support/$$lua $(DESTDIR)$(MODULES_DIR)/lua/$$lua ; \
 	done
 
+install-fq:	$(FQ_MODULES)
+	$(top_srcdir)/buildtools/mkinstalldirs $(DESTDIR)$(includedir)
+	for h in $(FQ_HEADERS); do \
+		$(INSTALL) -m 0444 $$h $(DESTDIR)$(includedir)/$$h ; \
+	done
```

Comment:
The headers are actually useful in the module's absence. As it exposed runtime resolving symbols (they are all inline functions)